pkg/addresses/addresses.go

```diff
@@ -33,15 +33,15 @@ type Book struct {
 // Get the addresses for the test environment.
 func GetTestBook() *Book {
 	return &Book{
-		CartesiDAppFactory:  common.HexToAddress("0x7122cd1221C20892234186facfE8615e6743Ab02"),
-		DAppAddressRelay:    common.HexToAddress("0xF5DE34d6BbC0446E2a45719E718efEbaaE179daE"),
-		ERC1155BatchPortal:  common.HexToAddress("0xedB53860A6B52bbb7561Ad596416ee9965B055Aa"),
-		ERC1155SinglePortal: common.HexToAddress("0x7CFB0193Ca87eB6e48056885E026552c3A941FC4"),
-		ERC20Portal:         common.HexToAddress("0x9C21AEb2093C32DDbC53eEF24B873BDCd1aDa1DB"),
-		ERC721Portal:        common.HexToAddress("0x237F8DD094C0e47f4236f12b4Fa01d6Dae89fb87"),
-		EtherPortal:         common.HexToAddress("0xFfdbe43d4c855BF7e0f105c400A50857f53AB044"),
-		InputBox:            common.HexToAddress("0x59b22D57D4f067708AB0c00552767405926dc768"),
-		CartesiDApp:         common.HexToAddress("0x70ac08179605AF2D9e75782b8DEcDD3c22aA4D0C"),
+		CartesiDAppFactory:  common.HexToAddress("0x610178dA211FEF7D417bC0e6FeD39F05609AD788"),
```

Comment:
does this need to be hard-coded?

Reply:
AFAIK, this is just to simplify our internal test/dev tool. It was a development decision. We may change it to no rely on hardcoded values in the future if needed be.

Reply:
It is hardcoded to simplify our lives, but ideally, this file should be generated automatically with `go generate`. Marcel, could you create an issue to tackle that?

Reply:
Issue #233 created